index.html

```diff
@@ -7,7 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles.css">
     <script src="challenge.js"></script>
-    <title>Challenge</title>
+    <title>Adam Smith: Porfolio</title>
   </head>
   <body>
     <h1>Adam Smith</h1>

```

Comment:
Kindly use Prettier or the default text format contained in the HTML mockup to properly arrange your code. This enables your code to be clean. 

thanks